services/src/main/java/org/exoplatform/officeonline/OfficeOnlineEditorService.java

```diff
@@ -681,10 +701,11 @@ protected void addFileURLProperties(Map<String, Serializable> map,
                                                              .append(PortalContainer.getCurrentRestContextName())
                                                              .toString();
 
-      // TODO: implement REST endpoint
       String downloadURL = new StringBuilder(platformRestURL).append("/officeonline/editor/content/")
-                                                             .append(WCMCoreUtils.getRemoteUser())
+                                                             .append(node.getUUID())
```

Comment:
Node UUID is a part of URL? Then it would be not required to encode it in the token.

Reply:
It's an outdated code. We pass accessToken instead of node UUID now.